synapse/tests/test_cmds_cron.py

```diff
@@ -288,20 +286,17 @@ async def getNextFoo():
                     await cmdr.runCmdLine('at --not-a-real-flag')
                     self.true(outp.expect('the following arguments'))
 
-                    await cmdr.runCmdLine('at {#foo} {#bar}')
```

Comment:
why is this test removed?

Reply:
Because we tightened up the arg definitions and it's no longer a useful test cast